packages/graphql-hooks/test/unit/GraphQLClient.test.js

```diff
@@ -345,16 +350,28 @@ describe('GraphQLClient', () => {
       it('sets body conforming to the graphql multipart request spec', () => {
         const actual = [...fetchOptions.body]
         const expected = [
-          ['operations', '{"query":"","variables":{"a":null}}'],
-          ['map', '{"1":["variables.a"]}'],
-          ['1', '[object Object]']
+          ['operations', '{"query":"","variables":{"a":null,"b":null}}'],
+          ['map', '{"1":["variables.a"],"2":["variables.b"]}'],
+          ['1', '[object Object]'],
+          ['2', stream]
         ]
         expect(actual).toEqual(expected)
       })
 
       it('does not set Content-Type header', () => {
         expect(fetchOptions.headers).not.toHaveProperty('Content-Type')
       })
+
+      it('throws if no FormData polyfill provided', () => {
+        delete global.FormData
```

Comment:
Could we replace it with an empty mock, and then re-instate the implementation after the test.

Reply:
I've decided it's better to split the tests for browser and Node JS upload. Browser test is now using `window.FormData` and "real" file instance.